lib/idl/idl_checkers.es6.js

```diff
@@ -65,5 +65,21 @@ module.exports = [
     logger.info('Checking top-level member names');
 
     eachKey(logger, left.members || [], right.members || [], 'name');
-  }
+  },
+  function rightMissingConstructor(logger, left, right) {
```

Comment:
Re: first comment: We do less work when `blink linked` is on the left, because the checker iterates over the left side to run checkers. In theory `blink linked` contains Blink-implemented spec'd APIs, whereas `blink` conatins ALL Blink-implemented APIs.

Re: second comment: Sure, we can use a count. Will fix.